### DISCUSSION TITLE
alt text on images?

### DISCUSSION THREAD

**Discussion by jaarong** [strapi/strapi#6414] [Legacy Topics]
Could someone point me to where the alt text that you enter on images is in graphql when you add that image to a another content type? Seems like it should be on the field for the image, but I can't find it in there.

**Comment by lauriejim** [comment]
Hello! If you are using the stable version of Strapi, you have now access to the Medial Lib.
By clicking on an image, you will be able to add these information in your media.
But for that you will have to use the stable version.

**Comment by willianjusten** [comment]
Hey @lauriejim , and how can I update the `alternativeText` from an image using GraphQL? 
As I can see by playing with the administration, we basically send a `POST` to `/upload?id=${ID_IMAGE}` with a `form-data` with some of the information.

With that, I managed to make it work with this endpoint with Rest API. But how can I do the same using GraphQL? I wasn't able to find any `mutations` that allows me to edit this part, only to upload a new image =/

Am I missing something?

**Reply by derrickmehaffy** [reply]
@willianjusten it would probably be better to open up a new discussion.
